ocsci/main.py

```diff
@@ -113,4 +113,7 @@ def main(arguments):
         '-p', 'ocsci.pytest_customization.marks',
         '-p', 'ocsci.pytest_customization.ocsci_logging',
     ])
+    from ocsci import config
+    from utility.utils import add_path_to_env_path
```

Comment:
Is it possible to do this import at the module level?

Reply:
Importing `config` module (or any module which import `config` module) before config initialization (`init_ocsci_conf()` - few lines before) breaks the configuration in the way, that some values wasn't available (https://github.com/red-hat-storage/ocs-ci/issues/233).

Reply:
Importing the config module isn't what causes the problem - it's code that runs at import time using values from it before it's initialized. Since this code executes *after* that, it will be fine.

Reply:
Hmm, maybe the problem you are referencing to was partly caused by using mutable objects as defaults, fixed in 8ed5847d8024fd828821000cc8fad6a9dcbfeac9

Reply:
That certainly seems plausible :)

Reply:
Fixed in f5ccedd.